xl/common.py

```diff
@@ -299,6 +299,27 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def platform_is_win() -> bool:
```

Comment:
I don't think we need these functions. On Windows `sys.platform` is always `win32` so we can just directly test for that. (And the other two functions are unused.)

Reply:
I would like to replace occurences of `sys.platform == 'darwin'` and `sys.platform == 'linux'.
But may be it is off topic.